src/lib/types/Product.ts

```diff
@@ -73,6 +77,7 @@ export interface Product extends Timestamps, ProductTranslatableFields {
 		hideContentBefore: boolean;
 		hideContentAfter: boolean;
 	};
+	hasLightVariations?: boolean;
```

Comment:
Does not need to be stored in my opinion - unless we want to add a way to disable variations while keeping them in DB (but this isn't the behavior at the moment, unticking the checkbox and saving the product will erase the variations)

We can rely on `variations.length` instead

Reply:
For now we don't need that but I think it can be useful in the future (that's a feeling, no precise plan yet).

Reply:
Rename to more generic `hasVariations` then

Reply:
Yeah, in the end, what won't be covered by "light" variations will be probably called "Product component" or "Product addon" or something like that. So let's go for hasVariations.